foursight_core/deploy.py

```diff
@@ -77,30 +77,43 @@ def build_config(cls, stage, trial_creds=None, trial_global_env_bucket=False,
                                'Need: S3_ENCRYPT_KEY, CLIENT_ID, CLIENT_SECRET, DEV_SECRET.'])
                       )
                 sys.exit()
-        for curr_stage in ['dev', 'prod']:
-            cls.CONFIG_BASE['stages'][curr_stage]['environment_variables']['S3_ENCRYPT_KEY'] = s3_enc_secret
-            cls.CONFIG_BASE['stages'][curr_stage]['environment_variables']['CLIENT_ID'] = client_id
-            cls.CONFIG_BASE['stages'][curr_stage]['environment_variables']['CLIENT_SECRET'] = client_secret
-            cls.CONFIG_BASE['stages'][curr_stage]['environment_variables']['DEV_SECRET'] = dev_secret
+        for curr_stage_name in ['dev', 'prod']:
+            curr_stage = cls.CONFIG_BASE['stages'][curr_stage_name]
+            curr_stage_environ = curr_stage['environment_variables']
+
+            curr_stage_environ['S3_ENCRYPT_KEY'] = s3_enc_secret
+            curr_stage_environ['CLIENT_ID'] = client_id
+            curr_stage_environ['CLIENT_SECRET'] = client_secret
+            curr_stage_environ['DEV_SECRET'] = dev_secret
             if env_name:
-                cls.CONFIG_BASE['stages'][curr_stage]['environment_variables']['ENV_NAME'] = env_name
+                curr_stage_environ['ENV_NAME'] = env_name
             if es_host:
-                cls.CONFIG_BASE['stages'][curr_stage]['environment_variables']['ES_HOST'] = es_host
+                curr_stage_environ['ES_HOST'] = es_host
             if trial_global_env_bucket:
                 # in the trial account setup, use a shorter timeout
-                cls.CONFIG_BASE['stages'][curr_stage]['lambda_timeout'] = 60
-                global_bucket = os.environ.get('GLOBAL_BUCKET_ENV')
-                if global_bucket:
-                    cls.CONFIG_BASE['stages'][curr_stage]['environment_variables']['GLOBAL_BUCKET_ENV'] = global_bucket
+                curr_stage['lambda_timeout'] = 60
```

Comment:
We should make this an argument as well. It should probably 10 (or 15) minutes (currently 60 seconds).

Reply:
I did this.